lib/order.rb

```diff
@@ -0,0 +1,104 @@
+require_relative 'customer'
+require 'csv'
+
+class Order
+
+  attr_reader :id
+  attr_accessor :products, :customer, :fulfillment_status
+
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id
+    @products = products
+    @customer = Customer.find(customer)
```

Comment:
I like the idea of calling `Customer.find` here. However, that means that the parameter `customer` is expected to be a customer ID. The tests for wave 1 pass in an instance of customer, which means these tests currently fail. You should either change your strategy here or go change the tests.